crt_portal/cts_forms/locale/zh_hant/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,2491 @@
+"\">call</a> to report a violation or report a violation by <a href="
+"\"#address-footer\">mail</a>."
+msgstr ""
+"如果您無法使用線上表格，您可以<a href=\"#phone-footer\">致電</a>或透過郵件舉"
+"報侵權事件。"
+
+#: templates/landing.html:336
+msgid "Already submitted a report?"
+msgstr "已經提交報告了嗎？"
+
+#: templates/landing.html:339
+msgid "Here's what to expect."
+msgstr "以下是後續流程。"
+
+#: templates/landing.html:343
+msgid ""
+"<strong>Thank you for your report.</strong> We carefully read each one to "
+"determine if we have the authority to help. We do our best to let you know "
+"about the outcome of our review. However, we may not always be able to "
+"provide you with updates because:"
+msgstr ""
+"<strong>感謝您提交報告。</strong>我們會仔細閱讀每份報告，以判定我們是否有權提"
+"供協助。我們會盡力通知您有關我們審查的結果。然而，我們可能不一定能夠為您提供"
+"進度更新，理由如下："
+
+#: templates/landing.html:345
+msgid ""
+"We're actively working on an investigation or case related to your report."
+msgstr "我們正在積極處理某項與您報告有關的案件。"
+
+#: templates/landing.html:346
+msgid "We're receiving and actively reviewing many reports at the same time."
+msgstr "我們同時收到許多報告且正在積極審查這些報告。"
+
+#: templates/landing.html:353
+msgid "Need urgent legal help?"
+msgstr "急需法律援助？"
+
+#: templates/landing.html:355
+msgid ""
+"Due to the amount of reports we receive, it can take several weeks for us to "
+"respond to your issue. Local legal aid offices or lawyers in your area may "
+"be able to quickly respond to or help with your concern"
+msgstr ""
+"基於我們所收到的報告量，我們可能需要幾週的時間才能回覆您的問題。您當地的法扶"
+"辦公室或律師可能可以迅速作出回覆或協助您處理您所擔憂的問題"
+
+#: templates/landing.html:356
+msgid ""
+"Contact Legal Services Corporation at <a aria-label=\"www.lsc.gov/find-legal-"
+"aid\" class=\"external-link--blue external-link--popup\" href=\"https://www."
+"lsc.gov/find-legal-aid\">lsc.gov/find-legal-aid</a> or call <a href="
+"\"tel:202-295-1500\">(202) 295-1500</a>."
+msgstr ""
+"請瀏覽 <a aria-label=\"www.lsc.gov/find-legal-aid\" class=\"external-link--"
+"blue external-link--popup\" href=\"https://www.lsc.gov/find-legal-aid\">lsc."
+"gov/find-legal-aid</a> 或致電 <a href=\"tel:202-295-1500\">(202) 295-1500</"
+"a> 與法律服務公司聯絡。"
+
+#: templates/landing.html:357
+msgid ""
+"Or visit <a class=\"external-link--blue external-link--popup\" aria-label="
+"\"www.findlegalhelp.org\" href=\"http://www.findlegalhelp.org/\">www."
+"findlegalhelp.org</a> or call <a href=\"tel:800-285-2221\">(800) 285-2221</"
+"a> to find a lawyer through the American Bar Association."
+msgstr ""
+"或請瀏覽 <a class=\"external-link--blue external-link--popup\" aria-label="
+"\"www.findlegalhelp.org\" href=\"http://www.findlegalhelp.org/\">www."
+"findlegalhelp.org</a>或致電 <a href=\"tel:800-285-2221\">(800) 285-2221</a> "
+"以透過美國律師協會尋找律師。"
+
+#: templates/landing.html:371
+msgid "Image attributions"
+msgstr "圖片來源"
+
+#: templates/partials/attachment-uploading-modal.html:8
+msgid "Uploading attachment..."
+msgstr ""
+
+#: templates/partials/attachment-uploading-modal.html:13
+msgid "Your attachment is uploading..."
+msgstr ""
+
```

Comment:
We should remove the `gettext` calls, so they're not included in `django.po` files, from these strings until we intend to translate them. At the moment these are only rendered on internal, English only, pages.